fixtures/AtomEmbedBlockElement.ts

```diff
@@ -0,0 +1,12 @@
+export const AtomEmbedBlockElement: AtomEmbedBlockElement = {
```

Comment:
This is now InteractiveAtomBlockElement - it'll need find and replace and updating in filenames etc!

```suggestion
export const InteractiveAtomBlockElement: InteractiveAtomBlockElement = {
```